.travis.yml

```diff
@@ -0,0 +1,14 @@
+sudo: false
+
+language: go
+go:
+  - 1.8
+  - tip
+
+before_install:
+  - go get github.com/golang/lint/golint
+
+script:
+  - make
+  - go vet
```

Comment:
I think you can integrate this in your Makefile, and just doing make here.
Something like,
```
vet:
	@echo ">> vetting code"
	@$(GO) vet $(pkgs)
```